### DISCUSSION TITLE
Help! Vercel stopped deploying. processor is not a function

### DISCUSSION THREAD

**Discussion by frankchau93** [vercel/community#463] [Help] [answered]
I'm running into issues deploying my vercel app. was working fine yesterday and now i have no idea why it stopped working. Here's the error:

Failed to compile.
--
11:57:20.481 |  
11:57:20.481 | HookWebpackError: processor is not a function
11:57:20.481 | at makeWebpackError (/vercel/path0/node_modules/next/dist/compiled/webpack/bundle5.js:45539:9)
11:57:20.482 | at /vercel/path0/node_modules/next/dist/compiled/webpack/bundle5.js:29467:12
11:57:20.482 | at eval (eval at create (/vercel/path0/node_modules/next/dist/compiled/webpack/bundle5.js:141008:10), <anonymous>:34:1)
11:57:20.482 | at runMicrotasks (<anonymous>)
11:57:20.482 | at processTicksAndRejections (internal/process/task_queues.js:95:5)
11:57:20.482 | -- inner error --
11:57:20.482 | TypeError: processor is not a function
11:57:20.482 | at /vercel/path0/node_modules/next/node_modules/cssnano-simple/src/index.js:46:22
11:57:20.482 | at Array.forEach (<anonymous>)
11:57:20.482 | at Object.module.exports [as default] (/vercel/path0/node_modules/next/node_modules/cssnano-simple/src/index.js:28:39)
11:57:20.482 | at CssMinimizerPlugin.optimizeAsset (/vercel/path0/node_modules/next/dist/build/webpack/plugins/css-minimizer-plugin.js:41:40)
11:57:20.482 | at /vercel/path0/node_modules/next/dist/build/webpack/plugins/css-minimizer-plugin.js:77:55
11:57:20.482 | at runMicrotasks (<anonymous>)
11:57:20.482 | at processTicksAndRejections (internal/process/task_queues.js:95:5)
11:57:20.482 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:74:20)
11:57:20.482 | at async Promise.all (index 0)
11:57:20.483 | at async /vercel/path0/node_modules/next/dist/build/webpack/plugins/css-minimizer-plugin.js:64:21
11:57:20.483 | caused by plugins in Compilation.hooks.processAssets
11:57:20.483 | TypeError: processor is not a function
11:57:20.483 | at /vercel/path0/node_modules/next/node_modules/cssnano-simple/src/index.js:46:22
11:57:20.483 | at Array.forEach (<anonymous>)
11:57:20.483 | at Object.module.exports [as default] (/vercel/path0/node_modules/next/node_modules/cssnano-simple/src/index.js:28:39)
11:57:20.483 | at CssMinimizerPlugin.optimizeAsset (/vercel/path0/node_modules/next/dist/build/webpack/plugins/css-minimizer-plugin.js:41:40)
11:57:20.483 | at /vercel/path0/node_modules/next/dist/build/webpack/plugins/css-minimizer-plugin.js:77:55
11:57:20.483 | at runMicrotasks (<anonymous>)
11:57:20.483 | at processTicksAndRejections (internal/process/task_queues.js:95:5)
11:57:20.483 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:74:20)
11:57:20.483 | at async Promise.all (index 0)
11:57:20.483 | at async /vercel/path0/node_modules/next/dist/build/webpack/plugins/css-minimizer-plugin.js:64:21

**Comment by JTaylor0196** [comment]
@frankchau93 Hey there! Would you be able to share your deployment URL with us?

**Reply by frankchau93** [reply]
@JTaylor0196 yes. here's a deployment URL: please help!

**Comment by frankchau93** [comment, marked as answer]
Here's the answer:

Add resolutions key to your package.json and put cssnano-preset-simple inside that should fix the problem

"resolutions": {

    "cssnano-preset-simple": "3.0.0"

  },
